From 238d6256fd47df2dd82e98812ef86a6dcbb2b212 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Wed, 20 Jan 2016 14:11:49 -0600
Subject: [PATCH 13/29] include LPDDR2 interleave and fix modes IMX6DQSCM

LPDDR2 interleave and fix modes included for IMX6DQSCM boards:
    - mx6dqscm qwks rev 1
    - mx6dqscm qwks rev 2
    - mx6dqscm evb 1gb
    - mx6dqscm evb 2gb

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 configs/mx6dqscm_1gb_evb_defconfig                           | 5 -----
 configs/mx6dqscm_1gb_evb_spinor_defconfig                    | 7 -------
 configs/mx6dqscm_1gb_fix_evb_defconfig                       | 5 +++++
 configs/mx6dqscm_1gb_fix_evb_spinor_defconfig                | 7 +++++++
 configs/mx6dqscm_1gb_fix_qwks_rev1_defconfig                 | 5 +++++
 configs/mx6dqscm_1gb_fix_qwks_rev1_spinor_defconfig          | 7 +++++++
 configs/mx6dqscm_1gb_fix_qwks_rev2_defconfig                 | 5 +++++
 configs/mx6dqscm_1gb_fix_qwks_rev2_spinor_defconfig          | 7 +++++++
 configs/mx6dqscm_1gb_interleaving_evb_defconfig              | 2 +-
 configs/mx6dqscm_1gb_interleaving_evb_spinor_defconfig       | 2 +-
 configs/mx6dqscm_1gb_interleaving_qwks_defconfig             | 5 -----
 configs/mx6dqscm_1gb_interleaving_qwks_rev1_defconfig        | 5 +++++
 configs/mx6dqscm_1gb_interleaving_qwks_rev1_spinor_defconfig | 7 +++++++
 configs/mx6dqscm_1gb_interleaving_qwks_rev2_defconfig        | 5 +++++
 configs/mx6dqscm_1gb_interleaving_qwks_rev2_spinor_defconfig | 7 +++++++
 configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig      | 7 -------
 configs/mx6dqscm_1gb_qwks_defconfig                          | 5 -----
 configs/mx6dqscm_1gb_qwks_spinor_defconfig                   | 7 -------
 configs/mx6dqscm_2gb_fix_evb_defconfig                       | 5 +++++
 configs/mx6dqscm_2gb_fix_evb_spinor_defconfig                | 7 +++++++
 configs/mx6dqscm_2gb_interleaving_evb_defconfig              | 2 +-
 configs/mx6dqscm_2gb_interleaving_evb_spinor_defconfig       | 2 +-
 22 files changed, 76 insertions(+), 40 deletions(-)
 delete mode 100644 configs/mx6dqscm_1gb_evb_defconfig
 delete mode 100644 configs/mx6dqscm_1gb_evb_spinor_defconfig
 create mode 100644 configs/mx6dqscm_1gb_fix_evb_defconfig
 create mode 100644 configs/mx6dqscm_1gb_fix_evb_spinor_defconfig
 create mode 100644 configs/mx6dqscm_1gb_fix_qwks_rev1_defconfig
 create mode 100644 configs/mx6dqscm_1gb_fix_qwks_rev1_spinor_defconfig
 create mode 100644 configs/mx6dqscm_1gb_fix_qwks_rev2_defconfig
 create mode 100644 configs/mx6dqscm_1gb_fix_qwks_rev2_spinor_defconfig
 delete mode 100644 configs/mx6dqscm_1gb_interleaving_qwks_defconfig
 create mode 100644 configs/mx6dqscm_1gb_interleaving_qwks_rev1_defconfig
 create mode 100644 configs/mx6dqscm_1gb_interleaving_qwks_rev1_spinor_defconfig
 create mode 100644 configs/mx6dqscm_1gb_interleaving_qwks_rev2_defconfig
 create mode 100644 configs/mx6dqscm_1gb_interleaving_qwks_rev2_spinor_defconfig
 delete mode 100644 configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig
 delete mode 100644 configs/mx6dqscm_1gb_qwks_defconfig
 delete mode 100644 configs/mx6dqscm_1gb_qwks_spinor_defconfig
 create mode 100644 configs/mx6dqscm_2gb_fix_evb_defconfig
 create mode 100644 configs/mx6dqscm_2gb_fix_evb_spinor_defconfig

diff --git a/configs/mx6dqscm_1gb_evb_defconfig b/configs/mx6dqscm_1gb_evb_defconfig
deleted file mode 100644
index 95cb836..0000000
--- a/configs/mx6dqscm_1gb_evb_defconfig
+++ /dev/null
@@ -1,5 +0,0 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-ldo.dtb\",SCMEVB"
-CONFIG_ARM=y
-CONFIG_TARGET_MX6DQSCMQWKS=y
-CONFIG_DM=y
-CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_evb_spinor_defconfig b/configs/mx6dqscm_1gb_evb_spinor_defconfig
deleted file mode 100644
index 6f3edac..0000000
--- a/configs/mx6dqscm_1gb_evb_spinor_defconfig
+++ /dev/null
@@ -1,7 +0,0 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-ldo.dtb\",SCMEVB"
-CONFIG_ARM=y
-CONFIG_TARGET_MX6DQSCMQWKS=y
-CONFIG_SYS_MALLOC_F=y
-CONFIG_SYS_MALLOC_F_LEN=0x400
-CONFIG_DM=y
-CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_fix_evb_defconfig b/configs/mx6dqscm_1gb_fix_evb_defconfig
new file mode 100644
index 0000000..92f8a3c
--- /dev/null
+++ b/configs/mx6dqscm_1gb_fix_evb_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-fix-ldo.dtb\",SCMEVB"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_fix_evb_spinor_defconfig b/configs/mx6dqscm_1gb_fix_evb_spinor_defconfig
new file mode 100644
index 0000000..c53de89
--- /dev/null
+++ b/configs/mx6dqscm_1gb_fix_evb_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-fix-ldo.dtb\",SCMEVB"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_fix_qwks_rev1_defconfig b/configs/mx6dqscm_1gb_fix_qwks_rev1_defconfig
new file mode 100644
index 0000000..bbc18eb
--- /dev/null
+++ b/configs/mx6dqscm_1gb_fix_qwks_rev1_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-fix-ldo.dtb\""
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_fix_qwks_rev1_spinor_defconfig b/configs/mx6dqscm_1gb_fix_qwks_rev1_spinor_defconfig
new file mode 100644
index 0000000..febf185
--- /dev/null
+++ b/configs/mx6dqscm_1gb_fix_qwks_rev1_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-fix-ldo.dtb\""
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_fix_qwks_rev2_defconfig b/configs/mx6dqscm_1gb_fix_qwks_rev2_defconfig
new file mode 100644
index 0000000..ed0a417
--- /dev/null
+++ b/configs/mx6dqscm_1gb_fix_qwks_rev2_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev2-fix-ldo.dtb\""
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_fix_qwks_rev2_spinor_defconfig b/configs/mx6dqscm_1gb_fix_qwks_rev2_spinor_defconfig
new file mode 100644
index 0000000..430327c
--- /dev/null
+++ b/configs/mx6dqscm_1gb_fix_qwks_rev2_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev2-fix-ldo.dtb\""
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_interleaving_evb_defconfig b/configs/mx6dqscm_1gb_interleaving_evb_defconfig
index b63ef1f..d2c7cb0 100644
--- a/configs/mx6dqscm_1gb_interleaving_evb_defconfig
+++ b/configs/mx6dqscm_1gb_interleaving_evb_defconfig
@@ -1,4 +1,4 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-ldo.dtb\",INTERLEAVING_MODE,SCMEVB"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-interleave-ldo.dtb\",INTERLEAVING_MODE,SCMEVB"
 CONFIG_ARM=y
 CONFIG_TARGET_MX6DQSCMQWKS=y
 CONFIG_DM=y
diff --git a/configs/mx6dqscm_1gb_interleaving_evb_spinor_defconfig b/configs/mx6dqscm_1gb_interleaving_evb_spinor_defconfig
index dcfea3a..f8574e6 100644
--- a/configs/mx6dqscm_1gb_interleaving_evb_spinor_defconfig
+++ b/configs/mx6dqscm_1gb_interleaving_evb_spinor_defconfig
@@ -1,4 +1,4 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-ldo.dtb\",INTERLEAVING_MODE,SCMEVB"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-evb-interleave-ldo.dtb\",INTERLEAVING_MODE,SCMEVB"
 CONFIG_ARM=y
 CONFIG_TARGET_MX6DQSCMQWKS=y
 CONFIG_SYS_MALLOC_F=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_defconfig
deleted file mode 100644
index 894734b..0000000
--- a/configs/mx6dqscm_1gb_interleaving_qwks_defconfig
+++ /dev/null
@@ -1,5 +0,0 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-ldo.dtb\",INTERLEAVING_MODE"
-CONFIG_ARM=y
-CONFIG_TARGET_MX6DQSCMQWKS=y
-CONFIG_DM=y
-CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_rev1_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_rev1_defconfig
new file mode 100644
index 0000000..a18ddfd
--- /dev/null
+++ b/configs/mx6dqscm_1gb_interleaving_qwks_rev1_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-interleave-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_rev1_spinor_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_rev1_spinor_defconfig
new file mode 100644
index 0000000..995766a
--- /dev/null
+++ b/configs/mx6dqscm_1gb_interleaving_qwks_rev1_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-interleave-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_rev2_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_rev2_defconfig
new file mode 100644
index 0000000..ae413b9
--- /dev/null
+++ b/configs/mx6dqscm_1gb_interleaving_qwks_rev2_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev2-interleave-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_rev2_spinor_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_rev2_spinor_defconfig
new file mode 100644
index 0000000..6e11060
--- /dev/null
+++ b/configs/mx6dqscm_1gb_interleaving_qwks_rev2_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev2-interleave-ldo.dtb\",INTERLEAVING_MODE"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig b/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig
deleted file mode 100644
index 58587a8..0000000
--- a/configs/mx6dqscm_1gb_interleaving_qwks_spinor_defconfig
+++ /dev/null
@@ -1,7 +0,0 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-ldo.dtb\",INTERLEAVING_MODE"
-CONFIG_ARM=y
-CONFIG_TARGET_MX6DQSCMQWKS=y
-CONFIG_SYS_MALLOC_F=y
-CONFIG_SYS_MALLOC_F_LEN=0x400
-CONFIG_DM=y
-CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_qwks_defconfig b/configs/mx6dqscm_1gb_qwks_defconfig
deleted file mode 100644
index 6e1b56f..0000000
--- a/configs/mx6dqscm_1gb_qwks_defconfig
+++ /dev/null
@@ -1,5 +0,0 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-ldo.dtb\""
-CONFIG_ARM=y
-CONFIG_TARGET_MX6DQSCMQWKS=y
-CONFIG_DM=y
-CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_1gb_qwks_spinor_defconfig b/configs/mx6dqscm_1gb_qwks_spinor_defconfig
deleted file mode 100644
index d3f6815..0000000
--- a/configs/mx6dqscm_1gb_qwks_spinor_defconfig
+++ /dev/null
@@ -1,7 +0,0 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=1024,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-1gb-qwks-rev1-ldo.dtb\""
-CONFIG_ARM=y
-CONFIG_TARGET_MX6DQSCMQWKS=y
-CONFIG_SYS_MALLOC_F=y
-CONFIG_SYS_MALLOC_F_LEN=0x400
-CONFIG_DM=y
-CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_2gb_fix_evb_defconfig b/configs/mx6dqscm_2gb_fix_evb_defconfig
new file mode 100644
index 0000000..143734b
--- /dev/null
+++ b/configs/mx6dqscm_2gb_fix_evb_defconfig
@@ -0,0 +1,5 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=2048,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-2gb-evb-fix-ldo.dtb\",SCMEVB,SCM_LPDDR2_2GB"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_2gb_fix_evb_spinor_defconfig b/configs/mx6dqscm_2gb_fix_evb_spinor_defconfig
new file mode 100644
index 0000000..59eee8f
--- /dev/null
+++ b/configs/mx6dqscm_2gb_fix_evb_spinor_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=2048,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-2gb-evb-fix-ldo.dtb\",SCMEVB,SCM_LPDDR2_2GB"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6DQSCMQWKS=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6dqscm_2gb_interleaving_evb_defconfig b/configs/mx6dqscm_2gb_interleaving_evb_defconfig
index 3635db4..daa5921 100644
--- a/configs/mx6dqscm_2gb_interleaving_evb_defconfig
+++ b/configs/mx6dqscm_2gb_interleaving_evb_defconfig
@@ -1,4 +1,4 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=2048,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-2gb-evb-ldo.dtb\",INTERLEAVING_MODE,SCMEVB,SCM_LPDDR2_2GB"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=2048,SYS_USE_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-2gb-evb-interleave-ldo.dtb\",INTERLEAVING_MODE,SCMEVB,SCM_LPDDR2_2GB"
 CONFIG_ARM=y
 CONFIG_TARGET_MX6DQSCMQWKS=y
 CONFIG_DM=y
diff --git a/configs/mx6dqscm_2gb_interleaving_evb_spinor_defconfig b/configs/mx6dqscm_2gb_interleaving_evb_spinor_defconfig
index 74ac50e..723bef2 100644
--- a/configs/mx6dqscm_2gb_interleaving_evb_spinor_defconfig
+++ b/configs/mx6dqscm_2gb_interleaving_evb_spinor_defconfig
@@ -1,4 +1,4 @@
-CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=2048,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-2gb-evb-ldo.dtb\",INTERLEAVING_MODE,SCMEVB,SCM_LPDDR2_2GB"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6dqscmqwks/imximage_scm_lpddr2.cfg,MX6Q,DDR_MB=2048,SYS_BOOT_SPINOR,DEFAULT_FDT_FILE=\"imx6dqscm-2gb-evb-interleave-ldo.dtb\",INTERLEAVING_MODE,SCMEVB,SCM_LPDDR2_2GB"
 CONFIG_ARM=y
 CONFIG_TARGET_MX6DQSCMQWKS=y
 CONFIG_SYS_MALLOC_F=y
-- 
2.7.4

